src/SphericalCollapse/spherical_collapse.cpp

```diff
@@ -85,6 +85,18 @@ template <> void RadhydroSimulation<CollapseProblem>::setInitialConditionsOnGrid
 	});
 }
 
+template <> void RadhydroSimulation<CollapseProblem>::createInitialParticles()
+{
+	// add particles at random positions in the box
+	bool generate_on_root_rank = true;
+	int iseed = 42;
+	int num_particles = 1000;
+	double particle_mass = 0.5 / static_cast<double>(num_particles);
```

Comment:
what should be the particle mass for a particular problem?

Reply:
like how would you set it for the pop iii test

Reply:
I tried to figure out what total mass I wanted in particles vs the gas (about equal), then divide by the number of particles. This will be different for a primordial gas cloud. There, it might be appropriate to choose the total gas mass relative to total particle+gas mass based on the universal baryon fraction $\Omega_b / \Omega_m$.